fish_passage_fieldwork.Rmd

```diff
@@ -0,0 +1,37 @@
+# Fish Passage Field Work Guide
+
+## Field Work Prep
+
+### Apps and Programs Used
+
+1. [Timestamp Camera](http://www.timestampcamera.com/) is used to take photos (landscape only) in the field. Time and utm settings can be found in Zotero with the citation key `irvine2021FishPassage`.
+
+2. [Avenza Maps](https://store.avenza.com/) is key for loading georeferenced pdf field maps from <https://hillcrestgeo.ca/outgoing/fishpassage/projects/>. We also add `kml` files of our target sites (taken from safety plans) to `Avenza` to make navigation easy. The paid version is necessary to load many maps at the same time for big projects.
+
+3. [Dropbox](https://www.dropbox.com/) is used for bulk uploads to `Avenza.`
+
+4. [Garmin BaseCamp](https://www.garmin.com/en-CA/software/basecamp/) is used to move `gpx` files in and out of our gps. We export tracks out of `Basemap` with the track names unchanged and we name the `gpx` files informatively (see screenshot below for example). **Note: The gps units have a little \$150 micro sd card behind the batteries that can come loose, this chip is key to knowing where we are going. Be careful when changing batteries.**
+
+
+### Study area research
+
+
+
+## Field Work 
+
+## Photos
+
+Taking proper photos is important for visualizing the site, data QA, and the quality of our report. See the [Apps and Programs Used] sections to download the TimeStamp app.  Below are some tips for taking proper photos:
+
+
+- Use the TimeStamp app
+- Ensure the current time and UTM coordinates in top right corner. 
+- Orient camera landscape when taking photos.
+- It is important to let the App “find” where you are before taking the first photos at a new site.  It can take a few seconds for the app to zoom into the location where you are standing once it is opened.
+- Closing the app every time you use it will help your phone battery not drain.
```

Comment:
Good segway for detail about phone and battery management.  Phone needs to be mounted on secure mount on dash so you can see the phone while driveing.  When driving phone gets plugged into jack in dash to charge. Alwayts carry external battery pack so that you can charge phone if it dies. phone always in a quality case

Need some sort of cloud backup so that photos sync to the cloud each night. they are on phone AND in mergin. At end of day take photos in order of all field pages

key to take photo of road with the site_id in the corner as first photo (or photo of site card filled in with site name, date, stream name) so that it is easy/possible to know which photos come from where.